themes/phil-coffe-theme/src/templates/portal_normal.ftl

```diff
@@ -0,0 +1,117 @@
+<!DOCTYPE html>
+
+<#include init />
+
+<html class="${root_css_class}" dir="<@liferay.language key="lang.dir" />" lang="${w3c_language_id}">
+
+<head>
+	<title>${the_title} - ${company_name}</title>
+	<link rel="stylesheet" type="text/css"
+      href="https://fonts.googleapis.com/css?family=Italianno">
+	<link rel="stylesheet" type="text/css"
+      href="https://fonts.googleapis.com/css?family=Dosis">
+	<meta content="initial-scale=1.0, width=device-width" name="viewport" />
+	<script src="http://cdn.alloyui.com/3.0.1/aui/aui-min.js"></script>
+
+
+	<@liferay_util["include"] page=top_head_include />
+</head>
+
+<body class="${css_class}">
+
+<@liferay_ui["quick-access"] contentId="#main-content" />
+
+<@liferay_util["include"] page=body_top_include />
+
+<@liferay.control_menu />
+
+<div class="container-fluid" id="wrapper">
+	<header id="banner" role="banner">
+		<div id="heading">
+			<script src="https://use.fontawesome.com/ee0a122bbd.js"></script>
```

Comment:
Font awesome should already be included in Liferay.  you should not need to include an additional script.